tests/console/journalctl.pm

```diff
@@ -289,7 +289,9 @@ sub cleanup {
 }
 
 sub post_fail_hook {
-
+    my ($self) = shift;
+    $self->SUPER::post_fail_hook;
+    # select_console('log-console');
```

Comment:
+1, done the same over here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12422/files :D. But let's go on with this one and I will later rebase. :) Thanks!